ui/src/error.ts

```diff
@@ -42,7 +73,11 @@ export const setFatal = (fatalError: FatalError): void => {
 };
 
 ipc.listenProxyError(proxyError => {
-  console.error("Proxy process exited", { proxyError });
+  log({
+    code: Code.UnexpectedProxyExit,
+    message: "Proxy process exicted unexpectedly",
```

Comment:
```suggestion
    message: "Proxy process exited unexpectedly",
```

Reply:
Might have been very excited when I wrote this.